File: src/main/java/Group2/Album/Collection/models/ArtistComment.java
package Group2.Album.Collection.models;

import javax.persistence.Entity;
import javax.persistence.ManyToOne;

import com.fasterxml.jackson.annotation.JsonIgnore;

@Entity
public class ArtistComment extends Comment {
	@ManyToOne
	@JsonIgnore
	private Artist artist;
	
	public ArtistComment() {
	}
	
	public ArtistComment(String content, Artist artist) {
		super(content);
		this.artist = artist;
	}
	
	public Artist getArtist() {
		return artist;
	}
}

File: src/main/resources/static/js/components/ArtistComment.js
export default function ArtistComment(comments) {
    return `
        <ul id="comment">
            ${comments.map(comment => {
        return `
                    <li class="comment">
                        <h3 class="commentContent">${comment.content}</h3>
                    </li>
                `;
    }).join('')}
        </ul>
        
        `} 